bin/docker-entrypoint

```diff
@@ -60,26 +60,18 @@ workers_healthcheck() {
 
 server() {
   # Recycle gunicorn workers every n-th request. See http://docs.gunicorn.org/en/stable/settings.html#max-requests for more details.
-  case $REDASH_PRODUCTION in
-    true)
-      echo "Starting Redash Server in production mode"
-      MAX_REQUESTS=${MAX_REQUESTS:-1000}
-      MAX_REQUESTS_JITTER=${MAX_REQUESTS_JITTER:-100}
-      TIMEOUT=${REDASH_GUNICORN_TIMEOUT:-60}
-      exec /usr/local/bin/gunicorn \
-        -b 0.0.0.0:5000 \
-        --name redash \
-        -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app \
-        --max-requests $MAX_REQUESTS \
-        --max-requests-jitter $MAX_REQUESTS_JITTER \
-        --timeout $TIMEOUT
-      ;;
-    *)
-      echo "Starting Redash Server in a dev mode"
-      export FLASK_DEBUG=1
-      exec /app/manage.py runserver --debugger --reload -h 0.0.0.0
-      ;;
-  esac
+  MAX_REQUESTS=${MAX_REQUESTS:-1000}
+  MAX_REQUESTS_JITTER=${MAX_REQUESTS_JITTER:-100}
+  TIMEOUT=${REDASH_GUNICORN_TIMEOUT:-60}
+  echo "Starting Redash Server"
+  exec /usr/local/bin/gunicorn \
+    -b 0.0.0.0:5000 \
+    --name redash \
+    -w${REDASH_WEB_WORKERS:-4} redash.wsgi:app \
+    --max-requests $MAX_REQUESTS \
+    --max-requests-jitter $MAX_REQUESTS_JITTER \
+    --timeout $TIMEOUT \
+    --reload
```

Comment:
@eradman added hotreload for gunicorn

Reply:
This should be in a separate PR

Reply:
Yeah, it does seem unrelated to the purpose of this PR. :smile:

@AndrewChubatiuk Please move the unrelated stuff (like this) to a separate PR.  More than one if needed.

Mixing stuff together just slows down the reviewing. :frowning:

Reply:
@eradman @justinclift moved this change to https://github.com/getredash/redash/pull/6940

Reply:
Thanks @AndrewChubatiuk. :smile: